ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -5,6 +5,11 @@
   loop: "{{ commands_before_kuttl_run }}"
   ignore_errors: true
 
+- name: 'Run make crc_storage_with_retries'
```

Comment:
If we run multiple kuttl tests, then crc storage might exist from before kuttl run. Can we first cleanup the existing crc storage then create the new one? It is just a suggestion not a blocker for this patch.

Reply:
ack, added cleanup